### PR TITLE
Update blacklist.txt

### PR DESCRIPTION
**Pull request by vuolter**

### CODE REVIEW

File: blacklist.txt [hunk 1]
```diff
@@ -15,13 +23,19 @@ hooks|Premium4Me.py
 hooks|ReloadCc.py
 hooks|Vipleech4uCom.py
 hoster|File4safeCom.py
+hoster|Keep2shareCC.py
+hoster|Keep2shareCc.py
```
Comment (stickell) [inline blacklist.txt]: Twice?

File: blacklist.txt [hunk 2]
```diff
@@ -15,13 +23,19 @@ hooks|Premium4Me.py
...
 hoster|MegaDebrid.py
 hoster|MegaNz.py
 hoster|MultiDebridCom.py
+hoster|PandaPlaNet.py
+hoster|PandaPlanet.py
```
Comment (stickell) [inline blacklist.txt]: Here too?

File: blacklist.txt [hunk 3]
```diff
@@ -1,12 +1,20 @@
+accounts|File4safeCom.py
+accounts|Keep2shareCc.py
 accounts|MegaDebrid.py
 accounts|MultiDebridCom.py
 accounts|NowVideoAt.py
 accounts|Premium4Me.py
 accounts|ShareRapidCom.py
+container|DLC_25.pyc
+container|DLC_26.pyc
+container|DLC_27.pyc
```
Comment (stickell) [inline blacklist.txt]: Why are you removing these?

File: blacklist.txt (hunk 1 above)
Comment (vuolter) [inline blacklist.txt]: No case sensitive here?

File: blacklist.txt (hunk 3 above)
Comment (vuolter) [inline blacklist.txt]: See https://github.com/pyload/pyload/commit/0dd0821ed115a0685388ac4477ae1bb5d5263c06

File: blacklist.txt (hunk 1 above)
Comment (stickell) [inline blacklist.txt]: Oh sorry, I haven't noticed the different case.

File: blacklist.txt (hunk 3 above)
Comment (stickell) [inline blacklist.txt]: Wow at last they have released a service to decrypt DLCs!

**Pull request merged**